src/openforms/js/lang/formio/en.json

```diff
@@ -1,4 +1,37 @@
 {
+  "select-key": "select",
```

Comment:
@SilviaAmAm can you do an English language double check? :grimacing:

Reply:
LGTM!